packages/hooks/use-format-byte/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { useFormatNumber } from "@yamada-ui/use-format-number"
+
+const bitPrefixes = ["", "kilo", "mega", "giga", "tera"]
+const bytePrefixes = ["", "kilo", "mega", "giga", "tera", "peta"]
+
+export interface FormatBytesOptions {
+  unit?: "bit" | "byte" | undefined
+  unitDisplay?: "long" | "narrow" | "short" | undefined
```

Comment:
I think we can just leave out undefined.
```suggestion
  unit?: "bit" | "byte" 
  unitDisplay?: "long" | "narrow" | "short" 
```

Reply:
Fixed.